.github/workflows/tests.yml

```diff
@@ -38,12 +38,6 @@ jobs:
       - name: Unit test
         run: pnpm run test
 
-      - uses: artiomtr/jest-coverage-report-action@v2.0-rc.1
```

Comment:
Is there some alternative to that action?

Reply:
@anbraten Not really, the next best thing is the setting to enforce 100% coverage that I enabled.